cms/templates/js/content-visibility-editor.underscore

```diff
@@ -12,9 +12,19 @@
         <li class="field-radio">
             <label class="label">
                 <input class="input input-radio" name="content-visibility" type="radio" value="hide_after_due" aria-described-by="hide_after_due_description">
-                <%- gettext('Hide content after due date') %>
+                <% if (self_paced) { %>
+                    <%- gettext('Hide content after course end date') %>
+                <% } else { %>
+                    <%- gettext('Hide content after due date') %>
+                <% } %>
             </label>
-            <p class='field-message' id='hide_after_due_description'> <%- gettext('After the subsection\'s due date has passed, learners can no longer access its content. The subsection remains included in grade calculations.') %> </p>
+            <p class='field-message' id='hide_after_due_description'>
+                <% if (self_paced) { %>
+                    <%- gettext('After the course\'s end date has passed, learners can no longer access subsection content.') %>
```

Comment:
I am suggesting to keep it for consistency with the other hide feature - otherwise, course teams may wonder whether there's a difference in grade calculations between the two hide features - when there's not.